### ISSUE TITLE
update-site with 3 different responses (always reproducible)

### ISSUE THREAD

**Issue by natt-tester** [GluuFederation/oxd#242]
**Prequisite:** a valid access token
**Steps:**
1. Make a `setup-client` call with the following body: 
```
{
  "authorization_redirect_uri": "https://client.example.org/cb",
  "op_host": "https://ce3.gluu.org"
}
```
2. Make an `update-site` call a valid token and the following body:
```
{
    "oxd_id":"6F9619FF-8B86-D011-B42D-00CF4FC964FF"
}
```
3. Make the same call 2 more times.

Result: The first response is 200 OK, the second -- 200 with an internal error, and the third one is a null command response with 500. This behavior is always reproducible with the exact same steps.

@yuriyz Shouldn't the response be the same upon the 2nd and 3rd update-site calls, and how can we make it better? **There's a null pointer in the logs, please check the last lines.**

![update_site_bug](https://user-images.githubusercontent.com/38302725/46799067-81719980-cd9f-11e8-9d52-fc5f0b6b6a16.gif)

<img width="562" alt="ok_response" src="https://user-images.githubusercontent.com/38302725/46798140-d7910d80-cd9c-11e8-86f5-fbba1a607d5e.png">
<img width="639" alt="internal_error_response" src="https://user-images.githubusercontent.com/38302725/46798158-e24ba280-cd9c-11e8-998a-fa57d8d1f3ab.png">
<img width="569" alt="null_response" src="https://user-images.githubusercontent.com/38302725/46798177-eecffb00-cd9c-11e8-93cb-f42499eb00be.png">

[oxd-server.log (1).tar.gz](https://github.com/GluuFederation/oxd/files/2468546/oxd-server.log.1.tar.gz)

**Comment by yuriyz** [comment]
It is bug. Fixed in 3.1.4.

**Comment by natt-tester** [comment]
It's fixed in new 3.1.4.
